upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1419,44 +1419,64 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 	}
 
 	// Set API server address to an IP from the cluster network CIDR
+	var controlPlaneIPs []string
```

Comment:
This looks like a bigger change, but is really saying "also use the fixed IP on hetzner"